### ISSUE TITLE
ARR: support award committee workflow for commitment venues

### ISSUE THREAD

**Issue by melisabok** [openreview/openreview-py#2266]
@celestemartinez was working on this workflow for ACL conference and so far it is working fine.

We need to make sure this is part of the workflow in openreview-py for commitment venues. 

Two aspects to consider:

- Authors must submit the anonymous version of the camera ready PDF.
- The camera ready PDF must not be visible to the award commitee.

**Comment by annargrs** [comment]
To clarify: not all authors will be submitting this extra anonymous version - only those who were notified to do so. So it's an optional upload.

**Comment by thamar-solorio** [comment]
Is this feature available then? can the EMNLP PCs make use of it?

**Comment by celestemartinez** [comment]
Yes, this is already available and we have done it for ACL. When you are ready to start this workflow, you can let us know and we can help you enable this.
